content/en/docs/concepts/services-networking/endpointslice.md

```diff
@@ -0,0 +1,108 @@
+---
+reviewers:
+- freehan
+title: Endpoint Slice
+feature:
+  title: Endpoint Slices
+  description: >
+    Scalable tracking of network endpoints in a Kubernetes cluster.
+
+content_template: templates/concept
+weight: 10
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+{{< glossary_definition term_id="endpointslice" length="short" >}}
+
+_Endpoint Slices_ provide a simple way to track network endpoints within a
+Kubernetes cluster. They offer a more scalable alternative to Endpoints
+resources within Kubernetes.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Motivation
```

Comment:
Overall, in concept-type documentation, I am wary of anchoring a page at a particular point in time. This section would be great for a blog post; in the concepts section, it might age.

I can imagine that this page look like this in many months' time, even after Endpoint Slices have become the norm for Service resources in Kubernetes.
I recommend keeping this section trim and moving it to the foot of the page.

If needed (to avoid a holdup), that could be something to tweak after the v1.16 release.

Reply:
That makes sense. I've shrunk this a bit and moved it to the bottom.